latest version (#4887)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/upgrade-dbt-version-in-cloud.md

---
title: "Upgrade dbt version in Cloud"
id: "upgrade-dbt-version-in-cloud"
---

In dbt Cloud, both [jobs](/docs/deploy/jobs) and [environments](/docs/dbt-cloud-environments) are configured to use a specific version of dbt Core. The version can be upgraded at any time.

## Environments

Navigate to the settings page of an environment, then click **Edit**. Click the **dbt version** dropdown bar and make your selection. You can select a previous release of dbt Core or [Keep on latest version](#keep-on-latest-version). Be sure to save your changes before navigating away.

> <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/choosing-dbt-version/example-environment-settings.png" width="90%" title="Example environment settings in dbt Cloud"/>

Reviewer: It's a little hard to see the "Keep on Latest Version" text. Might be beneficial to grab a screenshot with the text in the main bar rather than the v1.6.

Is there an environment to grab it from that also doesn't have the development view?

### Keep on latest version <Lifecycle status='beta' />

> By choosing to **Keep on latest version**, you always get the latest fixes and early access to new functionality for your dbt project. dbt Labs will handle upgrades for you, as part of testing and redeploying the dbt Cloud SaaS application.

Reviewer: Is the "early" part the selling point we want to make? With it, it could read like we're just forcing beta features into their environment, but alternatively, maybe it reads a little more like "you'll always have access to the newest functionality without having to select a version."

```suggestion
By choosing to **Keep on latest version**, you always get the latest fixes and immediate access to new features and functionality for your dbt project. dbt Labs will handle upgrades for you as part of testing and redeploying the dbt Cloud SaaS application.
```

You can upgrade to **Keep on latest version** no matter which version of dbt you currently have selected. As a best practice, dbt Labs recommends that you test the upgrade in development first; use the [Override dbt version](#override-dbt-version) setting to test _your_ project on the latest dbt version before upgrading your deployment environments and the default development environment for all your colleagues.

> 

Reviewer: Should we add language here about the ability to go from "Keep On Latest Version" back to a version and how ultimately that won't be possible?

Author: good flag! yes but not for this launch, not yet

:::tip Interested? Let us know!

> Currently, this feature is available in beta for select customers, rolling out to wider availability through February and March. Are you interested in getting early access? Please contact your account team to join!

Reviewer: ```suggestion
This feature is available in beta for select customers, rolling out to wider availability through February and March. If you're interested in early access, please contact your account team to join!
```
